API.yaml

```diff
@@ -1632,9 +1631,11 @@ components:
       type: object
       properties:
         index:
+          description: The rank of this line in the virtual table
```

Comment:
Looking at the Trace Compass, there exists both a rank and index for events. The protocol is not Trace Compass of course, but we need to make sure this is clear. It should be the position of this line in the complete virtual table, right?

Reply:
Rank is the term that is used in Trace Compass. In the protocol the don't need to use the term rank. It looks like that the index is used most when it comes to the TSP. So, to be consistent and clear let's use index everywhere in the TSP.